app/admin/legislations.rb

```diff
@@ -58,10 +56,9 @@
         attributes_table do
           row :title
           row :description
-          row :date_passed
           row :geography
           row 'Parent legislation', &:parent
-          row :sector
+          list_row 'Sectors', :laws_sector_links
```

Comment:
whatt? ;]

Reply:
not needed? =D

Reply:
Didn't know, that it could be in many sectors now.

Reply:
last time changes

Reply:
yes... super fun :/ basically they didn't do the classification and told it could be found from the text by matching with a list then it leads to multiple sectors of course ...

Reply:
:+1: